includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -620,11 +656,38 @@ static function() use ( $handles ) {
 	 * @param array $actions Actions, with action name as key and value being callback.
 	 */
 	public static function remove_actions( $actions = [] ) {
+		global $wp_filter;
+
 		foreach ( $actions as $action => $callbacks ) {
 			foreach ( $callbacks as $callback ) {
-				$priority = has_action( $action, $callback );
-				if ( false !== $priority ) {
-					remove_action( $action, $callback, $priority );
+				if ( is_array( $callback ) ) {
```

Comment:
Is checking for array correct here? A callback array could be just as valid for the `else` condition as well, such as in the case of removing an action for a static method. For example: `add_action( 'wp_head', [ 'FooClass', 'bar_method' ] );`

Reply:
Maybe it should be conditioned on an array having three arguments, to require the `$priority`?

Additionally, maybe the `else` condition should be moved up to `continue`. For exampe:

```php
$priority = has_action( $action, $callback );
if ( false !== $priority ) {
	remove_action( $action, $callback, $priority );
	continue;
}

list( $class, $method, $priority ) = $callback;
...
```

Reply:
Good catch. I've made some amendments in 9fcfff3c93ad6b624aad9316faaa4e75383150aa.